content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -66,7 +66,7 @@ This page shows you how to set up a simple Ingress which routes requests to Serv
 1. Create a Deployment using the following command:
 
     ```shell
-    kubectl create deployment web --image=gcr.io/google-samples/hello-app:1.0 --port=8080
+    kubectl create deployment web --image=gcr.io/google-samples/hello-app:1.0
```

Comment:
@radeksm the intent here is to have the Deployment named `web` listening on port 8080. The current command is wrong; the revision you suggest also does not set up a Deployment that listens inside the Pod on port 8080.

Maybe a `kubectl apply -f <url>` would work better here? What do you think?

Reply:
I think it might be OK to log a new issue to track the remaining defect, and merge these changes as they get things closer to correct.

Reply:
Also see https://github.com/kubernetes/kubernetes/pull/91113 which revises `kubectl` to allow this option!